deploy/olm-catalog/gitops-operator/manifests/gitops-operator.clusterserviceversion.yaml

```diff
@@ -96,9 +96,9 @@ spec:
         name: ''
         version: v1
   description: OpenShift GitOps is a layered product on top of OpenShift that enables teams to adopt GitOps principles for managing cluster configurations and automating secure and repeatable application delivery across hybrid multi-cluster Kubernetes environments. OpenShift GitOps is built around Argo CD as the core upstream project and assists customers to establish an end-to-end application delivery workflow on GitOps principles.
-  displayName: OpenShift Gitops
+  displayName: OpenShift GitOps Operator
```

Comment:
We wish to call it "OpenShift GitOps".

Reply:
Oh.  It should be called "Red Hat OpenShift GitOps" per @siamaksade .    I will change it.